src/cmds/scripts/pbs_habitat.in

```diff
@@ -512,22 +512,16 @@ upgrade_pbs_schema() {
 		if [ $ret -ne 0 ]; then
 			return $ret
 		fi
-	else
-		echo "Cannot upgrade PBS datastore version $ver"
-		ret=$?
-		if [ $ret -ne 0 ]; then
-			return $ret
-		fi
-		return 1
+		ver="1.2.0"
```

Comment:
In this block if we are taking care or both - v1.0.0 or v1.1.0 then can we remove the block of code from line 481 to 491 above which only works on 1.0.0? This seem redundant to me.